examples/golang/simple/filter.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/envoyproxy/envoy/contrib/golang/filters/http/source/go/pkg/api"
+)
+
+type filter struct {
+	callbacks api.FilterCallbackHandler
+	path      string
+}
+
+func (f *filter) sendLocalReply() api.StatusType {
```

Comment:
```suggestion
func (f *filter) sendLocalReplyForbidden() api.StatusType {
```

i have limited `go` experience so not sure if this suggestion is valid i just found `/localreply` etc a bit confusing at first and trying to think how we can work this into docs